gig-marketplace.md

```diff
@@ -0,0 +1,58 @@
+There simply is no single uniform way for contributors to discover all the InnerSource opportunities that exist that they might be interested in contributing to.
+Conversely, innerSource project owners also have no way of informing a general audience about their projects and its goals.
+
+## Context
+* There are multiple business units in your organization that do not communicate well.
+* Individuals have no way of discovering what InnerSource opportunities exist across the entire company.
+* InnerSource project owners have no way of advertising their project to potential contributors.
+* Individuals have been given some time by their management to get involved in InnerSource projects of their choosing.
+* Managers have no way of knowing what the true time/effort commitment will be if their employee chooses to take on tasks to support an InnerSource project.
+* There is no way of tracking or rewarding an individual's involvement in an InnerSource project.
+
+## Forces
+* Individuals are trapped within their business units/teams and do not often interact with those outside of them
+* Individuals are experiencing downtime on projects where they are underutilized
+* Managers are reluctant to allow involvement in work outside of their business unit priorities
+* Managers have no way of understanding what the time/effort commitment is for their employees wanting to get involved in an InnerSource project outside of their business unit
+* Managers have no way of tracking or rewarding their employee's involvement in InnerSource projects
+* There is no single portal through which InnerSource project owners can advertise the existence and goals of their projects
+* There is no single portal through which potential InnerSource contributors can discover project opportunities
+
+## Solutions
+Several separate initiatives were spawned across business units and then brought together by the InnerSource governance team to create an intranet based InnerSource marketplace.
+
+The first was an InnerSource project portal. This was an intranet site where InnerSource project owners could plug in the URL to their Git repo and (via the inclusion of an “innersource.json” file to their codebase) display small cards of information describing their projects. Each of these were then indexed and made searchable by end users based on a number of criteria including business unit name, technologies in use or trusted contributor names, etc.
+
+The second was a “gig” based intranet site where individuals could advertise their areas of expertise and project owners could list opportunities for collaboration describing exactly the skill and time requirements needed. To reward participants a points based system was attached to every action. Points accrued by accepting a gig are then captured and used as performance management criteria. All interactions are tracked via a public dashboard to measure and communicate the overall health of the marketplace.
+
```

Comment:
from @stephenmccall : it helped that the BU which started the portal had responsibility for similar type of systems/portals.